api-docs/paths/events/project-issues.json

```
@@ -46,6 +46,14 @@
           "type": "string"
         }
       },
+      {
+        "name": "hash",
+        "in": "query",
+        "description": "A list of hashes of groups to return. Is not compatible with 'query' parameter as it will only return the request groups with corresponding hashes. The maximum number of hashes that can be sent is 100. If more are sent, only the first 100 will be used.",
```

```suggestion
        "description": "A list of hashes of groups to return. Is not compatible with 'query' parameter. The maximum number of hashes that can be sent is 100. If more are sent, only the first 100 will be used.",
```

Given that it just errors out if you pass both, I think it's enough to say they're incompatible. My worry would be that saying "it only returns the groups that match the hashes" might lead people to think that we mean "of the groups the query would give you back, it only returns the groups that match the hashes," which is obv not true.

Might also be worth mentioning that a given group can have multiple hashes, so the number of groups returned may be smaller than the number of hashes sent (if two or more of the requested hashes point to the same group).